app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic
+
+GIT_COMMIT=54b2e307af763ce87bc8c88e80785d8114bf38dd
+
+DESCRIPTION="System information and benchmark tool for Linux systems"
+HOMEPAGE="https://github.com/lpereira/hardinfo"
+SRC_URI="https://github.com/lpereira/hardinfo/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk2 debug"
+
+RDEPEND="gtk2? ( >=x11-libs/gtk+-2.10:2 )
```

Comment:
consider dropping this and just forcing GTK 3

Reply:
Done